safehttp/mux.go

```diff
@@ -97,14 +97,38 @@ func NewServeMux(d Dispatcher, domains ...string) *ServeMux {
 
 // Handle registers a handler for the given pattern and method. If another
 // handler is already registered for the same pattern and method, Handle panics.
-func (m *ServeMux) Handle(pattern string, method string, h Handler) {
+//
+// Configs can be optionally passed in order to modify the behavior of the
+// interceptors on a registered handler. Passing a Config whose corresponding
+// Interceptor was not installed will produce no effect.
+func (m *ServeMux) Handle(pattern string, method string, h Handler, configs ...Config) {
+	interceps := map[string]Interceptor{}
+	for k, i := range m.interceps {
+		interceps[k] = i
+	}
+	var changed bool
+	for _, c := range configs {
+		for k, i := range interceps {
+			if i, ok := c.Apply(i); ok {
+				interceps[k] = i
+				changed = true
+			}
+		}
+	}
+	if !changed {
```

Comment:
This seems like a premature micro-optimization at first glance. What's the purpose of this?

Reply:
It's been removed in the latest implementation